### PR TITLE
[local-authentication] Fix isEnrolledAsync issue 

### PR DESCRIPTION
**Pull request by M1ST4KE**
# Why

Reported in issue #14118. According to documentation, everything works as expected, but the name and behavior of the function may mislead users, so it's better to provide new functions allowing users to explicitly decide if they want to check biometrics or any available authentication.

# How

Added new functions, allowing to check explicitly biometrics only or any authentication method.

# Test Plan

Checked in bare-expo app

# Checklist

- [x] Documentation is up to date to reflect these changes (eg: https://docs.expo.io and README.md).
- [x] This diff will work correctly for `expo build` (eg: updated `@expo/xdl`).
- [x] This diff will work correctly for `expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

**Review by Simek — COMMENTED** [review]
Currently, to see the docs changes on the website you need to manually regenerate the JSON data file using command:
```
et gdad -p expo-local-authentication
```

Also, since the feature is still in development it's nice to launch the docs website locally and verify in `unversioned` that the added parts are there and they are correctly rendered.

File: packages/expo-local-authentication/android/src/main/java/expo/modules/localauthentication/LocalAuthenticationModule.kt [hunk 1]
```diff
@@ -132,6 +132,22 @@ class LocalAuthenticationModule(context: Context) : ExportedModule(context), Act
     promise.resolve(result == BiometricManager.BIOMETRIC_SUCCESS)
```
Comment (lukmccall) [inline packages/expo-local-authentication/android/src/main/java/expo/modules/localauthentication/LocalAuthenticationModule.kt:132]: just call `isDeviceAuthenticationWithBiometricsEnrolledAsync`

**Review by lukmccall — CHANGES_REQUESTED** [review]
Looks good ✅

**Comment by lukmccall** [comment]
Fix CI

**Pull request closed without merging**